parcimonie.sh

```diff
@@ -155,11 +159,16 @@ getTimeToWait() {
 	# then we can encounter a modulo by zero. In this case, we use the following as fallback:
 	#   minimum wait time + rand(minimum wait time)
 	# = $minWaitTime + $(getRandom) % $minWaitTime
+        local waitTime
 	if [ "$(expr '2' '*' "$targetRefreshTime")" -le "$(getNumKeys)" ]; then
-		expr "$minWaitTime" '+' "$(getRandom)" '%' "$minWaitTime"
+		waitTime=$(expr "$minWaitTime" '+' "$(getRandom)" '%' "$minWaitTime")
 	else
-		expr "$minWaitTime" '+' "$(getRandom)" '%' '(' '2' '*' "$targetRefreshTime" '/' "$(getNumKeys)" ')'
+		waitTime=$(expr "$minWaitTime" '+' "$(getRandom)" '%' '(' '2' '*' "$targetRefreshTime" '/' "$(getNumKeys)" ')')
+	fi
+	if [ $waitTime -lt 0 ]; then
+		waitTime=$(expr "$waitTime" "*" "-1")
 	fi
+	echo $waitTime
 }
```

Comment:
Please be consistent to the file's rough style:
- Use `foo="$(bar)"` not `foo=$(bar)`
- Use `echo "$foo"` not `echo $foo`
- Use `[ "$foo" -lt 0 ]` not `[ $foo -lt 0 ]`
- Around strings without `$variables` in them, use `'` not `"`